src/DynamoManipulation/DynamoManipulation.csproj

```diff
@@ -38,7 +38,7 @@
     <Reference Include="PresentationFramework" />
     <Reference Include="ProtoGeometry, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>$(DynamoExternPath)\ProtoGeometry\ProtoGeometry.dll</HintPath>
+      <HintPath>..\..\extern\ProtoGeometry\ProtoGeometry.dll</HintPath>
```

Comment:
hmm weird. I think the original path was better but I guess that property was not set correctly?

Reply:
Yup somehow other projects are all using relative path except for this one and suddenly failed for me.. @aparajit-pratap was mentioning we will move to nuget eventually so this change is good for now